src/odemis/driver/picoquant.py

```diff
@@ -670,6 +681,13 @@ def _acquire(self):
                 # Wait until we have a start (or terminate) message
                 self._acq_wait_start()
 
+                # Open protection shutters
+                for det_name in self._detectors:
+                    logging.debug("Requesting shutter for %s to open." % det_name)
+                    shutter_name, ax_name, closed_pos, open_pos = self._shutter_axes[det_name]
+                    shutter = self._shutters[shutter_name]
+                    shutter.moveAbsSync({ax_name: open_pos})
```

Comment:
Don't use `moveAbsSync()`, which is blocking, but instead `moveAbs()`, which returns a Future. That way, you can start the movement on each axis, and eventually wait for all of them to be complete.
Note, if you want to be fancy, you can try to use `concurrent.futures.wait(fs)`.